Error: The `form` element was not requested.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: The prompt did not request a radio be checked by default.
diff --git a/output/Radio Group/3/257ffa95-23b6-447e-9e3b-330bf652f7e3.html b/output/Radio Group/3/257ffa95-23b6-447e-9e3b-330bf652f7e3.html
index 8173638..779edfa 100644
--- a/output/Radio Group/3/257ffa95-23b6-447e-9e3b-330bf652f7e3.html	
+++ b/output/Radio Group/3/257ffa95-23b6-447e-9e3b-330bf652f7e3.html	
@@ -1,14 +1,15 @@
-<form>
-  <fieldset>
-    <legend>Choose Theme:</legend>
-    <label>
-      <input type="radio" name="theme" value="light" checked> Light
-    </label>
-    <label>
-      <input type="radio" name="theme" value="dark"> Dark
-    </label>
-    <label>
-      <input type="radio" name="theme" value="high-contrast"> High Contrast
-    </label>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset>
+  <legend>Choose Theme:</legend>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light">
+    Light
+  </label>
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark">
+    Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast">
+    High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
